src/components/Preview.js

```diff
@@ -102,9 +103,22 @@ class Preview extends Component {
             }}
           />
         </div>
+        {gifProgress === 1 ? (
+          <div className="Preview-progress-success">
+            GIF ready for download!
+          </div>
+        ) : null}
       </div>
     );
   }
 }
 
 export default Preview;
+
+// gifFileName={this.props.gifFileName}
```

Comment:
I'm not a huge fan of these comments. I'm not entirely sure what they're for, and it's possible they're stale already? This is the only place that `defaultColor` shows up in the codebase. Let's just get rid of these.

Reply:
Done